pcli/src/opt.rs

```diff
@@ -123,9 +115,9 @@ impl Opt {
             let path = self.data_path.join(crate::VIEW_FILE_NAME);
             tracing::info!(%path, "using local view service");
 
-            let svc =
-                ViewService::load_or_initialize(path, fvk, self.node.to_string(), self.pd_port)
-                    .await?;
+            tracing::debug!("Connecting to URL: {}", &self.node);
+            let svc = ViewService::load_or_initialize(path, fvk, self.node.clone()).await?;
+            tracing::debug!("ALL DONE");
```

Comment:
non-blocking nit: here, rather than BEGIN/END annotations, it'd probably be better to use a tracing span, so that all the events that happen in the context of `load_or_initialize` are annotated with the contextual information. this could either be done by constructing a span manually and using the `Instrument` trait to wrap the future before it's awaited, or by using the `#[instrument]` macro on `ViewService::load_or_initialize` (probably using `skip` on the `fvk` parameter).